scripts/docs.js

```diff
@@ -0,0 +1,25 @@
+var cpdir = require('ncp').ncp;
```

Comment:
this version is fine. but perhaps use `shelljs` in the next version so it can consistently across Mac/Win/Linux.